### DISCUSSION TITLE
How does the cursor speed correlate to Windows cursor speed?

### DISCUSSION THREAD

**Discussion by ssun-cptmarvel** [linearmouse/linearmouse#121] [Q&A] [answered]
Would like to find the exact speed that correlates to the middle of the speed bar on windows if possible :)   6/11
Thanks for the awesome work.

**Comment by lujjjh** [comment, marked as answer]
There may not be a good way to do this for two closed-source operating systems.

You can try something out. On Windows, disable "Enhance pointer precision," and on macOS, set the acceleration to zero. Then, on Windows, measure how many pixels the mouse moves on the screen when you move it by one inch (using [DPI Analyzer](https://www.mouse-sensitivity.com/dpianalyzer/) or something similar), and do the same on Mac.

**Reply by ssun-cptmarvel** [reply]
Thank you. :)

**Reply by akfranco** [reply]
@ssun-cptmarvel Did you happen to find said sweet spot?

**Reply by ssun-cptmarvel** [reply]
Yup, using an online DPI measuring website, was basically spot on with the setting I felt best with

**Reply by simpygliz** [reply]
@ssun-cptmarvel To what speed did you change it to? The standard speed on Linear Mouse is 0.07. I use 800 DPI btw.

**Reply by lukerucker** [reply]
@ssun-cptmarvel what speed did you end up using?

**Reply by skareel** [reply]
@ssun-cptmarvel would like to know too, trying to match windows exactly for mouse feel

**Reply by danmarculet** [reply]
Wanted to know as well and tested it myself. For `1600dpi` the sweet spot seems to be `0.36`

**Comment by skareel** [comment]
For sure, I want the exact same mouse feel on mac as I have on windows, 6/11 setting.

**Reply by lukerucker** [reply]
I will do some tests later today (check this out if you haven't: https://www.mouse-sensitivity.com/dpianalyzer/). Using that tester indicated to me that 'disable mouse acceleration -1 mode' was giving me the expected 1600 DPI of my mouse. That said, one thing that makes this confusing is MacOS resolution settings. When I score my expected 1600 DPI score via mouse using this analyzer, am I moving a MacOS distance of 1600 dots or 3200 dots (MacOS retina 2x res vs native 1x res, I lean towards a true 1600 dots from my testing) when in 1512x982 mode on my 3024x1964 MBP14? In addition. We should independently test and see if we find the same results. One final note: '6/11' on Windows is just 5 out of 10 spaces if you look closely heh.

**Reply by skareel** [reply]
@lukerucker just did dpi analyzer on mac vs windows, came up with these scores.

Using logitech Ghub and mx518, my dpi is set to 600.

Mac: 

Actual counts: 1662
Actual Dpi: 332.40

Windows:

Actual counts: 1782
Actual DPI: 356.40

This might be caused by 2 different scaling settings though in mac vs windows. On windows, I use 117% scaling on a 1440p monitor. On mac, I'm using BetterDisplay with 90% scaling and I don't know what that scales to pixel wise. So with the same DPI of 600, the mouse on mac is slightly slower than windows but since they're slightly different scaling that might account for it I dont know. 

I just want to make sure that acceleration is COMPLETELY turned off, I can't stand it for one second.

**Reply by skareel** [reply]
@lukerucker I also clicked "Disable pointer acceleration" in linearmouse instead of setting the acceleration to -1. I don't want to have two sensitivity settings between logitech ghub and then linear mouse at the same time.

**Reply by lujjjh** [reply]
You may set the acceleration to 0 and adjust the speed until the movement speed is roughly identical with Windows. Then, test the actual DPI on macOS and Windows respectively. As the speed setting in LinearMouse is linear, you can easily calculate the factor and get the final speed value.

**Comment by HUMORCE** [comment]
> System: Sonoma 14.5

If you don't care about the speed while LinearMouse App is closing, go B.

A. **System settings**

1. Disable pointer acceleration, i.e.`"com.apple.mouse.linear" = 1`
2. Set tracking speed to 6/10, i.e. `"com.apple.mouse.scaling" = 1`

B. **LinearMouse App**

1. Disable pointer acceleration with LM App.
2. Set tracking speed to `1`.

C. **Mix them** (Recommended)

1. Finish system settings above
2. **Revert to system defaults** on **Pointer** panel of LM App then **Disable pointer acceleration** on same place.

`defaults read .GlobalPreferences | grep 'com.apple.mouse'`:
```
    "com.apple.mouse.linear" = 1;
    "com.apple.mouse.scaling" = 1;
```

I **think** the key `com.app.mouse.scaling` equivalent to `sensitivity`(multiplier) on FPS games or 6/11 mouse speed on Windows when it is `1`. So, if you want increase or decrease your eDPI, then set it lower or higher then `1`. And I **validated** it by measurements mention by lujjjh

**Reply by carlosresu** [reply]
So if I want Windows 6/11 400 DPI sensitivity on my Mac, I use LM Disable Pointer Acceleration and Tracking Speed 1 right?

**Reply by HUMORCE** [reply]
This is just equivalent to Windows 6/11. You can get it without LM, see part A.

For specific DPI, you need set default DPI of mouse first.

I have set mouse DPI to 400 and written to on-board memory with G Hub on Windows, I haven't installed any Logitech stuff on macOS and I don't want them. If your mouse doesn't support changing the on-board configuration, then you may need to install proprietary software, or adjust `com.app.mouse.scaling` to get 400 eDPI.
